scripts/clan_resources/freshkill.py

```diff
@@ -471,18 +510,30 @@ def feed_group(self, group: list, not_moon_feeding = False, queens = False, fed_
                 needed_amount = feeding_amount
             else:
                 if ration_prey and status == "warrior":
-                    feeding_amount = feeding_amount/2
+                    feeding_amount = feeding_amount / 2
 
-            if self.total_amount * 2 > self.amount_food_needed() and self.nutrition_info[cat.ID].percentage < 100:
+            if (
+                self.total_amount * 2 > self.amount_food_needed()
+                and self.nutrition_info[cat.ID].percentage < 100
+            ):
                 feeding_amount += 2
-            if self.total_amount * 1.8 > self.amount_food_needed() and self.nutrition_info[cat.ID].percentage < 100:
+            if (
+                self.total_amount * 1.8 > self.amount_food_needed()
+                and self.nutrition_info[cat.ID].percentage < 100
+            ):
                 feeding_amount += 1.5
-            elif self.total_amount * 1.2 > self.amount_food_needed() and self.nutrition_info[cat.ID].percentage < 100:
+            elif (
+                self.total_amount * 1.2 > self.amount_food_needed()
+                and self.nutrition_info[cat.ID].percentage < 100
+            ):
                 feeding_amount += 1
-            elif self.total_amount > self.amount_food_needed() and self.nutrition_info[cat.ID].percentage < 100:
+            elif (
+                self.total_amount > self.amount_food_needed()
+                and self.nutrition_info[cat.ID].percentage < 100
+            ):
                 feeding_amount += 0.5
 
-            if not_moon_feeding:
+            if not manual_feeding:
```

Comment:
Found the bug! Thank goodness for unit tests.

Reply:
Hooray!